pkg/localplan/watcher.go

```diff
@@ -245,18 +237,17 @@ func parsePositionData(positionData []byte) (NodePlanPosition, error) {
 }
 
 // Returns true if the plan needs to be applied, false if not
-// needsApplication, probeStatus, error
-func (w *watcher) needsApplication(planPosition NodePlanPosition, cp applyinator.CalculatedPlan) (bool, map[string]prober.ProbeStatus, error) {
+// needsApplication, probeStatus, error.
```

Comment:
this comment is no longer accurate as this function is no longer returning an error.